### DISCUSSION TITLE
Help with skill ranks and damage

### DISCUSSION THREAD

**Discussion by Dean304** [fh-fvtt/zweihander#173] [General]
On the actor sheet, I am unable to adjust skills or roll for damage in the chat window.

Please help.

Thanks,
Dean

**Comment by Re4XN** [comment]
Please describe what you are trying do to in _as much detail as possible_, including the system version and Foundry version. The easiest way to solve your problem quickly would be to contact us on Discord (Re4XN#8341 or kxfin#0458).
